definitions.yaml

```diff
@@ -136,7 +136,29 @@ definitions:
         type: array
         items:
           $ref: '#/definitions/V4NodeDefinition'
+      port_mappings:
+        type: array
+        description: |
+          Reveals the ports on the load balancer that are mapped to this cluster's ingress
+          and which protocol that port supports. Only shown and relevant on our on-prem KVM clusters.
+        items:
+          $ref: '#/definitions/V4PortMappingDefinition'
 
+  V4PortMappingDefinition:
+    type: object
+    properties:
+      ingress_port:
```

Comment:
I am not sure this should be exposed. This information is more or less useless because the customer cannot do anything with it AFAIK because this is for the k8s service manifest of the guest cluster which runs in our host cluster. The ports are also always the same. They will always be `30010` for HTTP and `30011` for HTTPS IIRC. See https://github.com/giantswarm/kubernetesd/blob/a790b6be50f18dfe1dd6f7fbd504e8ff2c333fc0/service/creator/ingress.go#L60-L103. 

The actual ingress-controller ports are configured somewhere else AFAIK.

Reply:
true, this is internal and always the same, also the customer can look this up inside their cluster, as this is basically their IC config